ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -728,13 +766,16 @@ const mapStateToProps = (state: GlobalState) => {
     ),
     profile: profileSelector(state),
     readServices: readServicesByIdSelector(state),
+    allSections: allServicesSectionsSelector(state),
```

Comment:
```suggestion
    allSections: [...localTabSections, ...allTabSections],
```
Is it right?

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1338/commits/9f7c7a2f2d8bb00b421f8928c593c28e554f1b9a